CliFx/Models/CommandSchema.cs

```diff
@@ -55,14 +55,16 @@ public override string ToString()
             if (!string.IsNullOrWhiteSpace(Name))
                 buffer.Append(Name);
 
-            foreach (var option in Options)
+            if (Options != null)
```

Comment:
`Options` shouldn't ever be null.

Reply:
That is true for normal library usage, however some unit tests were using test data such as: 

```csharp
yield return new TestCaseData(
	new []
	{
		new CommandSchema(null, "command1", null, null, null),
		new CommandSchema(null, "command2", null, null, null),
		new CommandSchema(null, "command3", null, null, null),
	},
	new CommandInput(new[] { "command4", "argument1" })
);
```

At least in my case that was preventing some test to run due to _Exception thrown during test SetUp_.

Arguably there are better ways to fix that starting with fixing the test data, but that was the easiest at the moment.

I can remove the null check if you want, fix the Stub or fix The command schema itself by coalescing null values to empty lists. Just let me know 😃

Reply:
Hmm, the test data seems wrong, I'll have to fix it. Let's ignore this in this PR then.